addons/web/static/src/legacy/js/views/basic/basic_model.js

```diff
@@ -2782,12 +2781,14 @@ var BasicModel = AbstractModel.extend({
                 context: list.getContext({withoutRecordData: true}) || {},
             });
         } else {
-            def = Promise.resolve(_.map(ids, function (id) {
-                return {id:id};
-            }));
+            def = Promise.resolve(
+                ids.map((id) => {
+                    return { id: id };
+                })
+            );
         }
         return def.then(function (result) {
-            var records = _.uniq(_.flatten(_.values(toFetch)));
+            var records = _.uniq(_.flatten(Object.values(toFetch || {})));
```

Comment:
looks like `_.flatten` is not necessary
```suggestion
            var records = unique(Object.values(toFetch || {}));
```

Reply:
it seems indeed